controller/backup_volume_controller.go

```diff
@@ -224,12 +224,28 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 	// Examine DeletionTimestamp to determine if object is under deletion
 	if !backupVolume.DeletionTimestamp.IsZero() {
 
+		isNeedCleanupRemoteData, err := isNeedToCleanupRemoteBackupData(backupVolume)
+		if err != nil {
+			return errors.Wrap(err, "failed to check if it needs to delete remote backup volume data")
+		}
+
+		if isNeedCleanupRemoteData {
+			deletingBackups, err := bvc.ds.ListBackupsWithBackupVolumeName(backupVolumeName)
+			if err != nil {
+				return errors.Wrap(err, "failed to list backups of the backup volume")
+			}
+			for _, backup := range deletingBackups {
+				if err = addBackupDeleteResourceOnlyLabel(bvc.ds, backup.Name); err != nil {
+					return errors.Wrapf(err, "failed to add deleting resource only label to backup %s", backup.Name)
+				}
+			}
```

Comment:
If `isNeedCleanupRemoteData` is `true`, what do we add the delete-custom-resource label to the backups?

Reply:
It's a typo. Modified, and moved it into the mutation of the `BackupVolume`.